### PR TITLE
Ftps transfers for SNAP application and docs

### PR DESCRIPTION
**Pull request by sree-cfa**
Transfer impl for SNAP application pdf and verification docs. 
---
Includes
- Generating meta data txt for docs
- Zipping up files
- PGP signing and encryption
- FTPS upload
- lots of placeholders

Excludes (will do in follow-up PR):
- schedule / chron job for transfers
- Reads/Updates from TBA Transmission table
- E2E Test using Transmission table info

### CODE REVIEW

File: src/main/java/org/ladocuploader/app/cli/PGPEncryptor.java [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+    lOut.write(data);
+    sGen.update(data);
+
+    lOut.close();
+    lGen.close();
+
+    sGen.generate().encode(cOut);
+    cOut.close();
+    cGen.close();
+    encGen.close();
```
Comment (bseeger) [inline src/main/java/org/ladocuploader/app/cli/PGPEncryptor.java:148]: does any of this write to disk and leave behind anything we should clean up?

File: src/main/java/org/ladocuploader/app/cli/SubmissionTransfer.java [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        } catch (Exception e) {
+          log.error("Error generating file collection for submission ID {}", submission.getId(), e);
+        }
+        zos.close();
+        baos.close();
+        docMeta.close();
+
+        // Encrypt and transfer
+        byte[] data = pgpEncryptor.signAndEncryptPayload(zipFileName);
+        ftpsClient.uploadFile(zipFileName, data);
```
Comment (bseeger) [inline src/main/java/org/ladocuploader/app/cli/SubmissionTransfer.java:98]: Should we delete the zipfile here, once it's in the byte array?   Mostly concerned about disk space filling up in the container.

File: src/main/java/org/ladocuploader/app/cli/PGPEncryptor.java (hunk 1 above)
Comment (sree-cfa) [reply src/main/java/org/ladocuploader/app/cli/PGPEncryptor.java:148]: It shouldn't but let me double check

Comment (sree-cfa) [reply src/main/java/org/ladocuploader/app/cli/PGPEncryptor.java:148]: All the streams are closed in this method or from the caller and doesnt write any local files so we should be good

File: src/main/java/org/ladocuploader/app/cli/SubmissionTransfer.java (hunk 2 above)
Comment (sree-cfa) [reply src/main/java/org/ladocuploader/app/cli/SubmissionTransfer.java]: Fixed

**Review by bseeger — APPROVED** [review]

**Pull request merged**